eu_dgc_v1_schema.yaml

```diff
@@ -43,12 +43,27 @@ properties:
               title: Identifier type
               description: The type of identifier (viz. VS-2021-04-08)
                 PPN = Passport
-                NNxxx = national Person Identifier (ISO 3166-1 alpha-3 country code)
-                CZ = Citizenship card
+                NN = National Person Identifier
+                CZ = Citizenship Card
                 HC = Health Card number
-                etc.
-              example: PPN
+                NI = National Unique Individual Identifier
+                MB = Member number
+                NH = National Helth Plan identifier
               type: string
+              enum:
+                - PPN
+                - NN
+                - CZ
+                - HC
+                - NI
+                - MB
+                - NH
+              example: NN
+            c:
+              title: Country
```

Comment:
Country should be optional, and if not available the country code given in the CWT should be assumed.

Reply:
I think it may have to be here as well to define the context of the identifier in question? It could be that country A issues a health certificate to a person with a passport issued by country B? And passport numbers are defined per issuing country. I suggest to actually make it mandatory.

Reply:
`c` is the country if the ID, not the DWG issuer - perhaps that need to be explicit?

Reply:
Yes, but my point was that a verifier steps into a verifying process knowing that country X has issued the DGC. Then we only need to inform that a passport is issued from country Y in the cases X != Y.

Reply:
Or just always include to be specific and put less magic on the verifier...